presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -101,6 +102,12 @@ void HttpServer::start(
   proxygen::HTTPServer::IPConfig cfg{
       httpAddress_, proxygen::HTTPServer::Protocol::HTTP};
 
+  if (SystemConfig::instance()->httpServerReusePort()) {
+    folly::SocketOptionKey portReuseOpt = {SOL_SOCKET, SO_REUSEPORT};
```

Comment:
What does SO_REUSEPORT actually do?

Reply:
+1. Can we explain a bit how is this option (https://lwn.net/Articles/542629/) and why batch process needs this option? Thanks!

Reply:
Yup I can add some comments to this property